osu.Game/OsuGame.cs

```diff
@@ -242,16 +242,19 @@ private bool globalHotkeyPressed(InputState state, KeyDownEventArgs args)
                     return true;
             }
 
-            if (state.Keyboard.ControlPressed)
+            if (currentScreen.UseSettingsOverlays)

```

Comment:
i'd rather not introduce arbitrary limits until we find a valid reason to do so.